recipe/meta.yaml

```diff
@@ -20,44 +20,46 @@ build:
 requirements:
   host:
     - python >=3.10,<3.13
-    - poetry-core >=1.0.0
-    - poetry-dynamic-versioning
+    - hatchling
+    - hatch-vcs
+    # - hatch-min-requirements
```

Comment:
I'm actually a bit confused by this. I would have thought we could simply use `pip` to install from PyPI.

Reply:
This is not allowed on conda.